### DISCUSSION TITLE
Ask me anything about Gatsby

### DISCUSSION THREAD

**Discussion by sidharthachatterjee** [github/india#20] [Open Source]
Hey everyone! Ask me anything about Gatsby here. Happy to answer questions, point to resources or hear feedback!

**Comment by S-ayanide** [comment]
Hey @sidharthachatterjee, Thanks for the thread. I have been using Gatsby for over 6 months now and had this query about having multiple fonts as a use-case for one of my projects. Is there a way to use multiple fonts in your project without installing any plugins! Instead by directly using CDNs? (Since Gatsby auto-generates the public folder every time it builds the index.html gets updated pretty frequently)

**Reply by sidharthachatterjee** [reply]
Hey @S-ayanide 

Thanks for the question! There's a simple fix for this. You can use a customised `html.js` in Gatsby to add your own script or link tags like the use case you describe. Read more about this at https://www.gatsbyjs.com/docs/custom-html/

That will ensure `index.html` has whatever you include in `html.js`
